exporter/v1_compatibility_test.go

```diff
@@ -115,3 +115,52 @@ func TestAddLocksMetrics(t *testing.T) {
 
 	assert.Equal(t, want, desc)
 }
+
+func Test_sumMetrics(t *testing.T) {
```

Comment:
let's rename this test
```suggestion
func TestSumMetrics(t *testing.T) {
```